unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Tests/FriendsHUDControllerShould.cs

```diff
@@ -57,6 +57,7 @@ public void SetUp()
             view = Substitute.For<IFriendsHUDComponentView>();
             view.FriendRequestCount.Returns(FRIEND_REQUEST_SHOWN);
             controller.Initialize(view);
+            controller.IsVisible = false;
```

Comment:
i dont like the idea of forcing the visibility state through an internal property.. you can call `SetVisibility(false)` in the `SetUp` and then clear the registered calls of mocks `view.ClearReceivedCalls()`. `IsVisible` should be private. That goes for the rest aswell